From c39f71f75f38e42adfcaf482c93596d41dd5a0a7 Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Sat, 9 May 2020 16:15:05 +0200
Subject: [PATCH] include <rtl/ustrbuf.hxx>

since otherwise it isn't resolvable in !OSL_BIGENDIAN case (like on s390x)

Change-Id: I18bc795744e70a459276b2c2d55e35411a5d672f
---
 sw/source/filter/ww8/WW8Sttbf.cxx | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/sw/source/filter/ww8/WW8Sttbf.cxx b/sw/source/filter/ww8/WW8Sttbf.cxx
index 54e05c42ac02..1e23c8ca89a7 100644
--- a/sw/source/filter/ww8/WW8Sttbf.cxx
+++ b/sw/source/filter/ww8/WW8Sttbf.cxx
@@ -28,6 +28,10 @@
 #include <sal/log.hxx>
 #include <osl/diagnose.h>
 
+#ifdef OSL_BIGENDIAN
+#include <rtl/ustrbuf.hxx>
+#endif
+
 namespace ww8
 {
     WW8Struct::WW8Struct(SvStream& rSt, sal_uInt32 nPos, sal_uInt32 nSize)
-- 
2.20.1

